work-item-link-type-blackbox_test.go

```diff
@@ -150,6 +152,19 @@ func (s *workItemLinkTypeSuite) TestCreateAndDeleteWorkItemLinkType() {
 	_ = test.DeleteWorkItemLinkTypeOK(s.T(), nil, nil, s.linkTypeCtrl, *workItemLinkType.Data.ID)
 }
 
+// TestCreateWorkItemLinkType tests if we can create the "test-epic-relates-userstory" work item link type
+func (s *workItemLinkTypeSuite) TestCreateAndDeleteWorkItemEpicUserStoryLinkType() {
```

Comment:
This test case is adding anything specific to Epic, right? How is this different from the previous one?